dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NotificationIT.java

```diff
@@ -131,4 +145,33 @@ public void testUpdateNotification() {
         assertEquals(currentMsg, message);  // confirm that the database entry was updated
 
     }
+    @Test
+    public void testLongNotification() throws IOException {
+
+        // set up a notification that is too long
+        Notification notification = longNotification();
+
+        // try to create notification with too long of a message
+        try {
+            curationApiAdmin.createNotification(notification);
```

Comment:
You need a `fail` after this line; otherwise if the test doesn't throw an error it will pass.